code/datums/diseases/advance/symptoms/radiation.dm

```diff
@@ -46,6 +46,7 @@
 /datum/symptom/radconversion
 	name = "Aptotic Culling"
 	desc = "The virus causes infected cells to die off when exposed to radiation, causing open wounds to appear on the host's flesh. The end result of this process is the removal of radioactive contamination from the host."
+	symptom_flags = SYMPTOM_DEAD_TICK_NECROTIC
```

Comment:
This one is probably better without dead processing considerings its activate functions